webapp/src/pages/reviewAssignment/components/ReviewAssignmentComponent.js

```diff
@@ -0,0 +1,187 @@
+import React, { Component } from "react";
+import { reviewService } from "../../../services/reviews";
+import { withRouter } from "react-router";
+import ReactTable from 'react-table'
+
+import 'react-table/react-table.css'
+
+import FormTextBox from "../../../components/form/FormTextBox";
+
+const DEFAULT_EVENT_ID = process.env.DEFAULT_EVENT_ID || 1;
```

Comment:
Please change this to REACT_APP_DEFAULT_EVENT_ID or it doesn't get passed in as env variable.